src/main/java/com/synopsys/integration/alert/provider/blackduck/descriptor/BlackDuckProviderUIConfig.java

```diff
@@ -54,7 +54,7 @@ public BlackDuckProviderUIConfig() {
     }
 
     private Collection<String> validateAPIToken(final FieldValueModel fieldToValidate, final FieldModel fieldModel) {
-        final String apiKey = fieldToValidate.getValue().orElse(null);
+        final String apiKey = fieldToValidate.getValue().orElse("");
         if (apiKey.length() < 64 || apiKey.length() > 256) {
```

Comment:
I would switch this to a `StringUtils.isBlank(apiKey)` and have the error say it's required. Otherwise validate the provided string.